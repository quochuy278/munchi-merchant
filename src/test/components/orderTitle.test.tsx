import React from "react";
import { render } from "@testing-library/react";
import { jest } from "@jest/globals";
import OrderTitle from "../../components/order/title";
import { create } from "react-test-renderer";

beforeEach(() => {
  jest.clearAllMocks();
});

describe("order Title component", () => {
  it("displays the OrderTitle", () => {
    // Render the component with some dummy user ID
    const component = create(
      <OrderTitle orderTitle="Pending" orderQuantity={7} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // Expect the component to have rendered the user's name
    // expect(OrderTitle).toHaveReturned(JSX.Element);
    expect(<OrderTitle orderTitle="Pending" orderQuantity={7} />)
      .toHaveBeenCalled;
  });
});
