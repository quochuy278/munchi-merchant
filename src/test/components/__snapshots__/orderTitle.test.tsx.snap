// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`order Title component displays the OrderTitle 1`] = `
<div
  className="section__title MuiBox-root css-15yyoo3"
>
  <h5
    className="MuiTypography-root MuiTypography-h5 css-1ivaqk5-MuiTypography-root"
  >
    Pending
  </h5>
  <div
    className="MuiBox-root css-q2ye7b"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 css-17893no-MuiTypography-root"
    >
      7
    </p>
  </div>
</div>
`;
